pkg/utils/transform/transform.go

```diff
@@ -116,7 +116,7 @@ func InjectLabel(key, value string, overwritePolicy OverwritePolicy, kinds ...st
 		labels[key] = value
 		err = unstructured.SetNestedStringMap(u.Object, labels, "metadata", "labels")
 		if err != nil {
-			return fmt.Errorf("error updateing labes for %s:%s, %s", kind, u.GetName(), err)
+			return fmt.Errorf("error updating labes for %s:%s, %s", kind, u.GetName(), err)
```

Comment:
also `labels` :D we were on a typo spree here

Reply:
Nice catch, I fixed it too